idc_index/index.py

```diff
@@ -883,7 +891,7 @@ def download_from_manifest(
         manifestFile: str,
         downloadDir: str,
         quiet: bool = True,
-        validate_manifest: bool = True,
+        validate_manifest: bool = False,
```

Comment:
I'll do some benchmarking and update this. My hypothesis is there will be no difference, as  validate_manifest: bool = False does nothing but suppress much-needed warning messages when something is unusual in a manifest. As I have said here..s5cmd does not stop even with an invalid URL.

https://github.com/ImagingDataCommons/idc-index/pull/56#discussion_r1584075999

Reply:
here's the benchmark for validating the entire NLST collection and entire IDC data. As you can see, there is no difference between validation=True and validation =False. Please let me know if you still would like to turn it off by default. 

I have added a few print statements to let the user know, we are parsing the manifest.

**NLST**:
![image](https://github.com/vkt1414/idc-index/assets/115020590/81573bbc-1038-4e09-8533-c6aeef6f8eb8)

**IDC**:
![image](https://github.com/vkt1414/idc-index/assets/115020590/8f027fe9-6c9d-470d-9020-8d4768402660)

Reply:
This looks great! (obviously!)